### ISSUE TITLE
Rufused To Connect

### ISSUE THREAD

**Issue by estar-app** [btcpayserver/btcpayserver-azure#5]
**After successful deploy to Azure, the app is refusing to connect whenever I searched the web using the given name in the dns. What could be the possible problem?**

**Comment by NicolasDorier** [comment]
What do you mean by refusing to connect? what do you see in your browser.

**Comment by estar-app** [comment]
This is what I see in the browser… Sent from Mail for Windows From: Nicolas DorierSent: Tuesday, May 17, 2022 9:58 AMTo: btcpayserver/btcpayserver-azureCc: estar-app; AuthorSubject: Re: [btcpayserver/btcpayserver-azure] Rufused To Connect (Issue #5) What do you mean by refusing to connect? what do you see in your browser.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by rustywave** [comment]
For anyone else seeing this, the site took a good 30 minutes to become available after the initial deploy. I was seeing the same "Refused to connect" error before.

**Comment by cf2013** [comment]
same problem, I followed the one click install for azure. every resource seems healthy but when using ip address or dns address or dnsadress/server/maintenance I see the same in the browser:

ERR_CONNECTION_REFUSED

It's been more than 30 min. 
I tried restarting the virtual machine.

**Comment by timgriffeth2024** [comment]
Has anyone resolved this? I'm facing the same issue after the one click azure deployment. Browser error, "the site can't be reached"

![image](https://github.com/user-attachments/assets/ae805564-c4e3-4ce9-ace1-42beb5e516cd)

**Comment by rustywave** [comment]
@timgriffeth2024 I have a reply earlier in this thread but I think you just need to wait a bit longer. For me it was 30 minutes.

**Comment by timgriffeth2024** [comment]
> @timgriffeth2024 I have a reply earlier in this thread but I think you just need to wait a bit longer. For me it was 30 minutes.

Thanks for replying. Waited overnight, and still the same error. It's been a couple of years since your post, maybe something is different in Azure now.

**Comment by rustywave** [comment]
Ah okay, yeah must be something else. I just saw someone else said their issue wasn't due to time either, hopefully they will respond. Sorry!

**Comment by timgriffeth2024** [comment]
> Ah okay, yeah must be something else. I just saw someone else said their issue wasn't due to time either, hopefully they will respond. Sorry!

Are you still running it in Azure? If so, I'm curious on how it's currently set up.

**Comment by timgriffeth2024** [comment]
Not sure what the deal was with the VM that was created from the template. I created my own VM, [installed BTCPay Server manually](https://github.com/btcpayserver/btcpayserver-docker/blob/master/README.md#full-installation-for-technical-users) and my server is running successfully.

I was browsing the initial VM created from the one click deployment, and it appeared that BTCPay server never got installed. So, that 1 click deployment template probably needs some updates.

**Comment by kuba-04** [comment]
I agree with @timgriffeth2024 , trying in 2025, seeing the same issue: This site can’t be reached
Log in to VM via SSH, nothing is running there on ports 80 or 433, there is no docker or anything. I though it would be too awesome to just start the server like that! I will try with manual install
